### DISCUSSION TITLE
How do I start using these dialogs?

### DISCUSSION THREAD

**Discussion by byronbytes** [ookii-dialogs/ookii-dialogs-winforms#33] [Q&A] [answered]
Call me stupid, but I can't find out how to make it open. I've tried   TaskDialog1.Show() but it doesn't work. (VB.nET)

**Comment by augustoproiete** [comment, marked as answer]
@byronbytes Take a look at the sample project that has an example of usage of every dialog:

https://github.com/ookii-dialogs/ookii-dialogs-winforms/tree/master/sample

You might also be interested in [this answer w/ some screenshots](https://github.com/ookii-dialogs/ookii-dialogs-winforms/discussions/18#discussioncomment-456347).

**Reply by byronbytes** [reply]
I found where my issue was 
![image](https://user-images.githubusercontent.com/53088136/126898913-ff1b49cc-c377-458b-9050-804ee2ebfda7.png)
and now I can add TaskDialogs! Thank you!
